os/fs/Kconfig

```diff
@@ -79,3 +79,10 @@ config SYSLOG_DEVPATH
 		(such as "/dev/ttyS1")/
 
 endif
+
+config GENERATE_FS_IMAGE
+	bool "Generate fs image"
+	default n
+	---help---
```

Comment:
1. Let's make a dependency with `FLASH_PARTITION`.
2. I don't know this location is proper or not. This is not the feature of smartfs itself. How about os/Kconfig? @Taejun-Kwon , Could you give me an opinion?
3. Could you describe about fs content? If no content added before build, empty smartfs image will be generated.

Reply:
1. Added.
2. I think it is related to fs, so I located in here.
3. Added.

Reply:
I added the dependency of `FS_SMARTFS`.